lntest/itest/lnd_test.go

```diff
@@ -2982,22 +2982,54 @@ func padCLTV(cltv uint32) uint32 {
 //
 // TODO(roasbeef): also add an unsettled HTLC before force closing.
 func testChannelForceClosure(net *lntest.NetworkHarness, t *harnessTest) {
-	success := t.t.Run("channelForceClosure", func(t *testing.T) {
-		ht := newHarnessTest(t, net)
+	// We'll test the scenario for all commitment types, to ensure outputs
+	// can be swept.
+	allTypes := []commitType{
+		commitTypeLegacy,
+		commitTypeTweakless,
```

Comment:
So the main change in this commit is that we are now also running the itest for tweakless?

Reply:
Maybe we don't want to do that, to keep the travis run time < 50 mins, if we aren't testing tweakless now either. Or just test tweakless and not legacy.

Reply:
Can confirm that tests run < 50 mins when leaving out one commit type.

Reply:
removed tweakless